app/com/linkedin/drelephant/tuning/hbt/MRConstant.java

```diff
@@ -0,0 +1,55 @@
+package com.linkedin.drelephant.tuning.hbt;
+
+final class MRConstant {
+  static final String MAPPER_TIME_HEURISTIC = "Mapper Time";
+  static final String MAPPER_SPEED_HEURISTIC = "Mapper Speed";
+  static final String MAPPER_MEMORY_HEURISTIC = "Mapper Memory";
+  static final String MAPPER_SPILL_HEURISTIC = "Mapper Spill";
+  static final String REDUCER_TIME_HEURISTIC = "Reducer Time";
+  static final String REDUCER_MEMORY_HEURISTIC = "Reducer Memory";
+
+  static final String MAPPER_MEMORY = "mapreduce.map.memory.mb";
+  static final String MAPPER_HEAP = "mapreduce.map.java.opts";
+  static final String REDUCER_MEMORY = "mapreduce.reduce.memory.mb";
+  static final String REDUCER_HEAP = "mapreduce.reduce.java.opts";
+  static final String MAP_SPLIT_SIZE = "mapreduce.input.fileinputformat.split.maxsize";
+  static final String PIG_SPLIT_SIZE = "pig.maxCombinedSplitSize";
+  static final String NUMBER_OF_REDUCER = "mapreduce.job.reduces";
+  static final String BUFFER_SIZE = "mapreduce.task.io.sort.mb";
+  static final String SPILL_PERCENTAGE = "mapreduce.map.sort.spill.percent";
+
+  static final String MAX_VIRTUAL_MEMORY = "Max Virtual Memory (MB)";
+  static final String MAX_PHYSICAL_MEMORY = "Max Physical Memory (MB)";
+  static final String MAX_TOTAL_COMMITTED_HEAP_USAGE = "Max Total Committed Heap Usage Memory (MB)";
+  static final String AVERAGE_TASK_INPUT_SIZE = "Average task input size";
+  static final String AVERAGE_TASK_RUNTIME = "Average task runtime";
+  static final String NUMBER_OF_TASK = "Number of tasks";
+  static final String RATIO_OF_SPILLED_RECORDS_TO_OUTPUT_RECORDS = "Ratio of spilled records to output records";
+  static final String SORT_BUFFER = "Sort Buffer";
+  static final String SORT_SPILL = "Sort Spill";
+
+  static final double VIRTUALMEMORY_TO_PHYSICALMEMORY_RATIO = 2.1;
+  static final double HEAPSIZE_TO_MAPPERMEMORY_SAFE_RATIO = 0.75;
+  static final double AVG_TASK_TIME_LOW_THRESHOLDS_FIRST = 1.0;
```

Comment:
Are these task_time thresholds defined in configuration, if not how did you come up with these values? Same comment for other kind of threshold and "increase decrease" values.